prometheus-to-sd/translator/translator.go

```diff
@@ -482,9 +482,31 @@ func getMonitoredResourceFromLabels(config *config.CommonConfig, labels []*dto.L
 	}
 
 	resourceLabels := config.MonitoredResourceLabels
+	if _, found := resourceLabels["project_id"]; !found {
+		resourceLabels["project_id"] = config.GceConfig.Project
+	}
+	if _, found := resourceLabels["cluster_name"]; !found {
+		resourceLabels["cluster_name"] = config.GceConfig.Cluster
+	}
+	if _, found := resourceLabels["location"]; !found {
+		resourceLabels["location"] = config.GceConfig.Zone
+	}
+
+	// When MonitoredResource type is not "k8s_*", default "instance_id" label to GCE instance name.
+	if prefix != "k8s_" {
```

Comment:
Shouldn't this be:
```
if prefix == "" {
```

Reply:
This if loop is the default for types other than "k8s_*".
And it should use numeric instance id.
For "k8s_node", it should use instance name.